src/types/LintState.ts

```diff
@@ -0,0 +1,85 @@
+import { action, extendObservable } from "mobx";
+import { Diagnostic } from "./Diagnostic";
+import { Lintable } from "./Lintable";
+
+export enum LINT_STATE {
```

Comment:
tweak for [style guide](https://docs.canva.tech/common/style-guides/typescript/#enums)

```suggestion
export enum LintState {
```